packages/toast/src/ToastContainer/ToastContainer.styles.ts

```diff
@@ -0,0 +1,243 @@
+import { TransitionStatus } from 'react-transition-group';
+import { mix } from 'polished';
+
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing, transitionDuration } from '@leafygreen-ui/tokens';
+
+import { TOAST_CONSTANTS } from '../constants';
+import { toastBGColor } from '../InternalToast';
+
+// Keeping this to ease future debugging
```

Comment:
I like the idea of keeping it for future debugging - it would be nice to have some documentation on how to make use of it for future maintainers